models/model.api.js

```diff
@@ -1,6 +1,7 @@
 const db = require('../db/connection');
 const data = require('../db/data/test-data');
 const fs =require('fs/promises');
+const {getArticlesInDateOrderQuery} = require('./query-strings');
```

Comment:
I have never seen anyone do this before 😄 To be clear that isn't a bad thing at all. I was just a little confused how you made this model function SO short!